### ISSUE TITLE
Release it

### ISSUE THREAD

**Issue by gavinking** [hibernate/query-validator#26]
I think we should do a release of this project to Maven. WDYT @emmanuelbernard @FroMage @Sanne ? Should we do it? Precisely *how* can I put jars in Maven? This is something I have never once done before...

**Comment by Sanne** [comment]
Yes we should release it. I can take care of that, but not this week..

Had already started to polish the build from committing jars in the repo, should finish that first too.

**Comment by gavinking** [comment]
One question: which version of the JDK should we be targeting here? Because I've been retesting everything and noticed that it doesn't compile on JDK 13 due to some minor changes to APIs in `tools.jar`. I'm going to test on JDK 11 now.

**Comment by Sanne** [comment]
Quarkus still needs JDK8 compatibility today. 

Hopefully we'll soon target JDK11 only but we're not there yet. Testing with JDK 12, 13, 14 would be nice too.

**Comment by gavinking** [comment]
> Quarkus still needs JDK8 compatibility today.

OK, great, perfect, we'll leave `query-validator` targeting JDK8.

**Comment by pumpkinlink** [comment]
is the project abandoned?

**Comment by olsavmic** [comment]
Hi @gavinking, I believe releasing the tool on maven central would help adoption a lot - we decided NOT TO use the tool just because it complicates the CI test configuration.

And hopefully, an increased adoption would be a driver for further development (perhaps even by the community).
Is there any specific reason this package was never officially released?

---

Thank you all for the great work you put into Hibernate / JPA! :)
